Makefile

```diff
@@ -40,6 +41,15 @@ lint-go: .install.golangci-lint
 verify-dependencies: $(GOTOOLS_BINDIR)/zeitgeist
 	$(GOTOOLS_BINDIR)/zeitgeist validate --local-only --base-path . --config dependencies.yaml
 
+.PHONY: verify-prettier
+verify-prettier: prettier ## Run prettier on the project.
```

Comment:
No support for `make help` yet, I believe. :smile: I suppose, you plan to add it here, too.